exercises/finaleworkflow/README.md

```diff
@@ -0,0 +1,49 @@
+# Exercise 4: Finale Workflow
+
+One of the benefits of Temporal is that it provides SDKs for several
+languages and you can use multiple languages in the context of a single
+Workflow. For example, you might write your Workflow in TypeScript but use
+Java or Go for an Activity in that workflow.
+
+The last exercise in this workshop does exactly that. The Workflow
+itself is written in TypeScript, but the Activity that is executed as part
+of this Workflow is written in Java, as is the Worker that runs it.
+Since the Activity is written in Java, it's able to use a Java graphics
+library that would otherwise be would be incompatible with a typical
+TypeScript program.
+
+# Start the Worker (Java)
+
+In one terminal, run the following command from the `finaleworkflow` directory:
+
+```
+java -classpath \
```

Comment:
I would recommend omitting the backslashes and newlines here, putting everything on one line. This course will likely have a high proportion of Windows users and I don't think their command prompts support this UNIX style of line continuation.

```
java -classpath java-activity-and-worker.jar io.temporal.training.PdfCertWorker
```